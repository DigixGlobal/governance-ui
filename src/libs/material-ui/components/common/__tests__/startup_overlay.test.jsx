import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import StartupOverlay from '../startup_overlay';

describe('<StartupOverlay />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <StartupOverlay />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
