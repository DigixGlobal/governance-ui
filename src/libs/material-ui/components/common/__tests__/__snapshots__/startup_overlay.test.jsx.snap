// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StartupOverlay /> renders correctly 1`] = `
<div>
  <Label
    color="purple"
    content="Beware of phishing! Always check the URL!"
    icon="warning sign"
    pointing={true}
    style={
      Object {
        "left": 10,
        "maxWidth": "100%",
        "position": "fixed",
        "top": 0,
        "zIndex": 1002,
      }
    }
  />
  <EZModal
    actions={[Function]}
    closeOnDimmerClick={false}
    content={
      <div
        style={
          Object {
            "background": "rgba(0,0,0,.05)",
            "maxHeight": "50vh",
            "overflowY": "scroll",
            "padding": "1em",
          }
        }
      >
        <p>
          TODO: Default TOS
        </p>
      </div>
    }
    header="Spectrum"
    initiallyOpen={true}
    onClose={[Function]}
  />
</div>
`;
